studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx

```diff
@@ -1,28 +1,246 @@
-import { DisplayApiSettings, DisplayConfigSettings } from 'components/ui/ProjectSettings'
+import Link from 'next/link'
 import { FC } from 'react'
-import GetStartedPanel from './GetStartedPanel'
+import SVG from 'react-inlinesvg'
+import { useRouter } from 'next/router'
+import { Button, IconKey, IconArchive, IconExternalLink, IconCode } from '@supabase/ui'
+
+import Panel from 'components/ui/Panel'
+import APIKeys from './APIKeys'
+import GetStartedHero from './GetStartedHero'
 
 interface Props {}
 
-const NewProjectPanel: FC<Props> = ({}) => (
-  <div className="grid grid-cols-12 gap-8">
-    <div className="col-span-12">
-      <GetStartedPanel />
-    </div>
-    <div className="col-span-4">
-      <div className="space-y-2">
-        <h3 className="text-xl text-scale-1200">Connecting to your new project</h3>
+const NewProjectPanel: FC<Props> = ({}) => {
+  const router = useRouter()
+  const { ref } = router.query
+
+  return (
+    <div className="grid grid-cols-12 gap-20">
```

Comment:
I forget: do we want an lg:grid on these so they collapse on mobile, or do we want the horizontal scroll?

Reply:
ahh yeap - i'll make them all into a column for mobile